package.json

```diff
@@ -174,5 +174,8 @@
   "browser": {
     "./dist/react-select.cjs.js": "./dist/react-select.browser.cjs.js",
     "./dist/react-select.esm.js": "./dist/react-select.browser.esm.js"
+  },
+  "resolutions": {
+    "csstype": "^3.0.2"
```

Comment:
This is necessary because we still have some dependencies on the website that use Emotion 10 which depends on csstype 2 which clashes with the csstype 3 types and produces build errors.